crates/miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -917,7 +917,7 @@ end
 #!
 #! Panics if:
 #! - the note index is greater or equal to the total number of input notes.
-#! - is_active_note is 1 and no input note is not being processed (attempted to access note inputs
+#! - is_active_note is 1 and no input note is not being processed (attempted to access note storage
 #!   from incorrect context).
```

Comment:
It seems like we overseen this previously: this line (and all similar ones) should explain the error reason according to the specific error used in the proc body. For example, in that case we use `ERR_NOTE_ATTEMPT_TO_ACCESS_NOTE_ASSETS_WHILE_NO_NOTE_BEING_PROCESSED`, so this line should use `assets` instead of `storage`.

```suggestion
#! - is_active_note is 1 and no input note is not being processed (attempted to access note assets
#!   from incorrect context).
```